terraform_templates/iam_template.py

```diff
@@ -0,0 +1,23 @@
+from typing import Mapping
+
+
+def terraform_iam_template(config: Mapping):
+    dict_template = {
+        "terraform": {
+            "required_providers": {
+                "aws": {
+                    "source": "hashicorp/aws",
+                    "version": "~> 3.00"
+                }
+            }
+        },
+        "provider": {
+            "aws": {
+                "access_key": config["aws"]["provider"]["access_key"],
+                "secret_key": config["aws"]["provider"]["secret_key"],
```

Comment:
It should not be necessary to include both `secret_key` and `access_key` in the policy.

Reply:
I see that this is old functionality, so this should probably be future work to remove it.

Reply:
Reached out to Erich for his thoughts on authentication, we decided on using stored aws profiles instead.

Changed config to have ```shared_credentials_file``` and ```profile``` fields instead of ```access_key``` and ```secret_key```